site/app/templates/forum/CreatePost.twig

```diff
@@ -60,8 +60,12 @@
                     {% endif %}
                 </a>
             {% endif %}
-        {% endif %}
+        {% endif %} 
 
+        <button class="upduck-button text-decoration-none key_to_click" tabindex="0" onClick="toggleLike({{post.id}}, {{post.thread_id}}, '{{current_user}}')">
```

Comment:
I'm not sure I like where the upduck is positioned as its between a whole bunch of other buttons and I think it would look better if it were off to one side. Maybe a discussion item.